server/lib/adapters/changes.ex

```diff
@@ -6,24 +6,52 @@ require Protocol
 defmodule Realtime.Adapters.Changes do
   defmodule(Transaction, do: defstruct([:changes, :commit_timestamp]))
 
-  defmodule(NewRecord,
-    do: defstruct([:type, :record, :schema, :table, :columns, :commit_timestamp])
-  )
+  defmodule NewRecord do
+    @derive {Jason.Encoder, except: [:is_rls_enabled, :users]}
+    defstruct [
+      :columns,
+      :commit_timestamp,
+      :schema,
+      :table,
+      :record,
+      :users,
+      :type,
+      is_rls_enabled: true
```

Comment:
as `is_rls_enabled` is the first check to determine if a database change should be broadcast to a user topic, defaulting to `true` and up to `Replication` and `RLS.ReplicationPoller` to set to `false`